### PR TITLE
Add pre-recording checklist

### PR DESCRIPTION
**Pull request by garricn**
In response to #27, this is the first of many future PRs to add this kind of documentation.

### CODE REVIEW

File: Resources/pre-recording-checklist.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Define Intros - Define the introduction for each host (e.g., "Our first host is Garric Nahapetian. Garric is an iOS Enginner at Tinder as well as the founder of Learn Swift City.")  
+- Define Audience - Ensure each host knows who the target audience is for the episode
+- Set Limits: Ask each host what they definitely DO and definitely do NOT want to talk about
+
+## Last Call
+- Allow each host one last chance to use the restroom or get a drink of water
+- Ask each host to turn their electronics on airplane and or slient mode and the computer they are using to record on do not disturb
+- Ensure that any phone or cellular device including Apple Watches are either on airplace mode or away from microphones
+- Remind each host that Its just a conversation, no need to apologize
+- Remind each host to try to not use no curse words
```
Comment (BasThomas) [inline Resources/pre-recording-checklist.md]: To not use* instead of “to not use no”

Comment (JohnSundell) [reply Resources/pre-recording-checklist.md]: Or simply "Remind each host to try not to curse" 🙂

Comment (garricn) [reply Resources/pre-recording-checklist.md]: 🤦🏽‍♂️

Comment (BasThomas) [reply Resources/pre-recording-checklist.md]: All good, no worries @garricn :)

**Pull request merged**